department-of-veterans-affairs/caseflow
# Justin/caseflow 4604 rpo completed tab

Resolves [CASEFLOW-4604](https://vajira.max.gov/browse/CASEFLOW-4604)

### Description
Created a completed tab for the RPO Queue and created Spec tests to insure correct functionality

### Acceptance Criteria
- [ ] Determination of the behavior and triggers of the Completed tab
- [ ] When a user of the RPO organization, sends an appeal back to the EMO organization, it is removed from its previous tab and shows up on the RPO’s Completed tab.
- [ ] When a user of the RPO organization, sends an appeal to BVA Intake (for docketing), it is removed from its previous tab and shows up on the RPO’s Completed tab.

### Testing Plan
1. Create an appeal with Education issues and choose "require predocketing"
2. Log in as an EMO user, and assign the appeal to an RPO
3. Log in as an RPO User, set the task as "in progress", and then complete the task
4. The task should now show in the completed tab for the next 7 days.

### User Facing Changes
![image](https://user-images.githubusercontent.com/95875751/165604359-37de393d-04d7-467a-b4a5-b9b60ba0abd0.png)

### [Link to Designs](https://bahdigital.invisionapp.com/share/QMIAIA7TANS#/screens/296070625)

### Code Changes
![github com_department-of-veterans-affairs_caseflow_compare_feature_CASEFLOW-2738 Justin_CASEFLOW-4604-rpo-completed-tab_expand=1](https://user-images.githubusercontent.com/95875751/165617430-218776c1-84ab-45cc-aa45-3d440872a9c2.png)

### Code Documentation Updates
